tools/lambda-promtail/lambda-promtail/cw.go

```diff
@@ -41,6 +41,9 @@ func parseCWEvent(ctx context.Context, b *batch, ev *events.CloudwatchLogsEvent)
 }
 
 func processCWEvent(ctx context.Context, ev *events.CloudwatchLogsEvent) error {
+
+	fmt.Println("processing new CWEvent\n")
```

Comment:
`Println()` already adds a newline
```suggestion
	fmt.Println("processing new CWEvent")
```